src/docs/development/cli.mdx

```diff
@@ -4,7 +4,21 @@ import { Divider } from '../../app/components';
 
 # PX Blue CLI
 
-The PX Blue CLI (Command-Line Interface) is a tool for quickly starting a new project using PX Blue. It will automatically create a skeleton project in your chosen framework, add Material and PX Blue component library dependencies, include and integrate PX Blue themes, typography, and colors, and optionally add PX Blue code formatting and linting profiles.
+The PX Blue CLI (Command-Line Interface) is a tool for quickly starting new projects using PX Blue configurations.
```

Comment:
```suggestion
The PX Blue CLI (Command-Line Interface) is a tool for quickly starting new projects using PX Blue. It will create a skeleton project in your chosen framework and automatically integrate:

- Material and PX Blue component libraries
- PX Blue themes
- Open Sans Typography package, 
- PX Blue colors package
- PX Blue code formatting and linting profiles (optional)
```